firestore/src/android/field_value_android.h

```diff
@@ -87,7 +90,10 @@ class FieldValueInternal {
   std::vector<FieldValue> array_value() const;
   MapFieldValue map_value() const;
 
-  const jni::Global<jni::Object>& ToJava() const { return object_; }
+  jni::Local<jni::Object> ToJava() const {
```

Comment:
nit: Move the `ToJava()` implementation into the .cc file, now that it is not a simple "getter" method.